doc/jekyll_tools/css/manual.css

```diff
@@ -0,0 +1,75 @@
+.search-box-container {
```

Comment:
`manual.css` is used only for the manual of the C core on the homepage so it's not needed here.

Reply:
ok